processing/src/test/java/io/druid/query/datasourcemetadata/DataSourceMetadataQueryTest.java

```diff
@@ -81,7 +81,7 @@ public void testContextSerde() throws Exception
                                                         "useCache",
                                                         true,
                                                         "populateCache",
-                                                        true,
+                                                        "true",
```

Comment:
What do you mean?

Here I just changed one of the test values, to make sure that "getContextValue" returns strings/booleans as-is but "getContextBoolean" turns them all into booleans.